### PR TITLE
Blog post for the OBS 2.9 release

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: _posts/releases/2018-03-08-release_of_obs_2.9.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+There is the new [image template page](http://openbuildservice.org/2017/05/11/new-image-templates-page/)
+where you can branch images from and base your work on.
+With the [kiwi editor](http://openbuildservice.org/help/manuals/obs-best-practices/cha.obs.best-practices.webuiusage.html#kiwi_editor_how_to)
+you can further configure your image, add packages and whatnot. All this is part of the
+[SUSE Studio Express](http://openbuildservice.org/2017/09/27/suse-studio-express/) we started last year.
+With the recently added [EC2 cloud upload]() You can directly upload your EC2 image from OBS to the Amazon Web Services (AWS).
+
+We revamped our notification system, including RSS Feeds for user's notifications,
+[RabbitMQ support](http://openbuildservice.org/2018/02/12/rabbitmq-integration/),
+a nicer UI for the notifications page and much more.
```
Comment (bgeuken) [inline _posts/releases/2018-03-08-release_of_obs_2.9.md]: @adrianschroeter @mlschroe I was thinking of mentioning here one or two more things we did. Any backend stuff you want to have added?

File: _posts/releases/2018-03-08-release_of_obs_2.9.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of the Open Build Service, Version 2.9
+category: releases
+author: OBS Team
+---
+
+The Build Service Team is happy to announce the release of Open Build Service 2.9!
```
Comment (Ana06) [inline _posts/releases/2018-03-08-release_of_obs_2.9.md]: A blog post with only one emoji.... :sob: 

`The Build Service Team is happy to announce the release of Open Build Service 2.9! :tada:
`

File: _posts/releases/2018-03-08-release_of_obs_2.9.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Among many bug fixes this release comes with a big bunch of features.
+There is the new [image template page](http://openbuildservice.org/2017/05/11/new-image-templates-page/)
```
Comment (Ana06) [inline _posts/releases/2018-03-08-release_of_obs_2.9.md]: `There is the new` sound strange... What about `Thanks to the new`?  :thinking:

File: _posts/releases/2018-03-08-release_of_obs_2.9.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+where you can branch images from and base your work on.
+With the [kiwi editor](http://openbuildservice.org/help/manuals/obs-best-practices/cha.obs.best-practices.webuiusage.html#kiwi_editor_how_to)
+you can further configure your image, add packages and whatnot. All this is part of the
```
Comment (Ana06) [inline _posts/releases/2018-03-08-release_of_obs_2.9.md]: and whatnot? :confused:

Comment (Ana06) [reply _posts/releases/2018-03-08-release_of_obs_2.9.md]: part of SUSE Studio Express, without `the`

File: _posts/releases/2018-03-08-release_of_obs_2.9.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+[SUSE Studio Express](http://openbuildservice.org/2017/09/27/suse-studio-express/) we started last year.
+With the recently added [EC2 cloud upload]() You can directly upload your EC2 image from OBS to the Amazon Web Services (AWS).
```
Comment (Ana06) [inline _posts/releases/2018-03-08-release_of_obs_2.9.md]: Again `With the`... What about `Last but not least, the recently added EC2 cloud upload feature, allows you to directly...`?

And you forgot the link in EC2 cloud upload.. or do you want to link the blog post that is not published yet? :thinking:

File: _posts/releases/2018-03-08-release_of_obs_2.9.md (hunk 2 above)
Comment (bgeuken) [reply _posts/releases/2018-03-08-release_of_obs_2.9.md]: :sob: True, but it's just a first draft :construction:

File: _posts/releases/2018-03-08-release_of_obs_2.9.md (hunk 3 above)
Comment (bgeuken) [reply _posts/releases/2018-03-08-release_of_obs_2.9.md]: Much better. Let me change it

File: _posts/releases/2018-03-08-release_of_obs_2.9.md (hunk 4 above)
Comment (bgeuken) [reply _posts/releases/2018-03-08-release_of_obs_2.9.md]: I intend to change the whatnot, but don't have a better phrase yet.

Regarding the `the`. I wish we would have a native speaker. It sounds better to me with the `the` :thinking:

File: _posts/releases/2018-03-08-release_of_obs_2.9.md (hunk 5 above)
Comment (bgeuken) [reply _posts/releases/2018-03-08-release_of_obs_2.9.md]: I didn't forgot the link. This depends on #125

**Comment by bgeuken** [comment]
@Ana06 Updated. Now with 5 smilies :1st_place_medal:

**Pull request merged**